PyInstaller/exceptions.py

```diff
@@ -12,3 +12,18 @@
 
 class ExecCommandFailed(SystemExit):
     pass
+
+
+class HookError(Exception):
+    """base class for hook related errors"""
```

Comment:
```suggestion
    """Base class for hook related errors."""
```

Reply:
Hi @bwoodsend - I see you suggested this change, and then accepted this pull request. Do you want me to make this change and then push again?

Reply:
Yes please.